yarn.lock

```diff
@@ -3703,6 +3712,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@types/react@npm:types-react@beta":
```

Comment:
You should use `resolutions` to enforce a single version of `@types/react` and `@types/react-dom` as suggested in the [upgrade guide](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#installing)

Multiple versions of `@types/react` is not supported just like multiple versions of `react` are not supported.